src/Vektonn.DataSource/Kafka/KafkaConsumerConfig.cs

```diff
@@ -20,5 +21,17 @@ public KafkaConsumerConfig(string[] bootstrapServers)
         public TimeSpan MaxRetryDelay { get; set; } = TimeSpan.FromSeconds(10);
         public TimeSpan WatermarkOffsetsQueryTimeout { get; set; } = TimeSpan.FromSeconds(1);
         public int ConsumeBatchSize { get; set; } = 10_000;
+        public Action<ConsumerConfig> ConsumerConfigUserTuning { get; set; } = config => {};
```

Comment:
Naming suggestions (for this property and method below):
* `UserTuning` looks like `Customisation`, `Tuning` looks like `Initialisation`;
* They should be verbs, not nouns (e.g. `CustomiseConsumerConfig` and `InitialiseConsumerConfig`).

Reply:
Thank you for suggestions! Changed names in conversation above

Reply:
https://github.com/vektonn/vektonn/commit/520265f1b6939e76f9dbc612e4fa4394e9776dd8
Renamed methods and delegates